### PR TITLE
Add HSA actuals to API

### PR DESCRIPTION
**Pull request by smcclure17**
This PR addresses https://trello.com/c/mQAfsaWF/399-add-cdc-community-level-related-metrics-to-api

Snapshot 3132 was built off of this PR and can be previewed at:
https://data.covidactnow.org/snapshot/3132/v2/county/25017.timeseries.json
https://data.covidactnow.org/snapshot/3132/v2/state/MA.timeseries.json
https://data.covidactnow.org/snapshot/3132/v2/cbsa/11100.timeseries.json

https://data.covidactnow.org/snapshot/3132/v2/state/MA.timeseries.csv
https://data.covidactnow.org/snapshot/3132/v2/county/MA.timeseries.csv

Unrelated, but black seems to be failing due to https://github.com/psf/black/issues/2964, I've started looking to fix in #1277

### CODE REVIEW

File: tests/libs/build_api_v2_test.py [hunk 1]
```diff
@@ -92,12 +92,21 @@ def test_build_summary_for_fips(
...
                 "currentUsageTotal": nyc_latest["hospital_beds_in_use_any"],
             },
+            hsaHospitalBeds={
+                "capacity": nyc_latest.get("staffed_beds_hsa"),
+                "currentUsageCovid": nyc_latest.get("current_hospitalized_hsa"),
+                "currentUsageTotal": nyc_latest.get("hospital_beds_in_use_any_hsa"),
+            },
             icuBeds={
                 "capacity": nyc_latest["icu_beds"],
-                "totalCapacity": nyc_latest["icu_beds"],
```
Comment (smcclure17) [inline tests/libs/build_api_v2_test.py:97]: This extra field was in the test, not sure, why (it doesn't exist in the API) so I removed it

File: api/can_api_v2_definition.py [hunk 2]
```diff
@@ -121,6 +121,19 @@ class Actuals(base_model.APIBaseModel):
...
+ * capacity - Current staffed acute bed capacity.
+ * currentUsageTotal - Total number of acute beds currently in use
+ * currentUsageCovid - Number of acute beds currently in use by COVID patients.
+""",
+    )
+    hsaHospitalBeds: Optional[HospitalResourceUtilization] = pydantic.Field(
+        ...,
+        description="""
+Information about acute bed utilization details aggregated for the corresponding
+Health Service Area (HSA). For more on HSAs see:
```
Comment (mikelehen) [inline api/can_api_v2_definition.py:134]: Maybe say something about it only being available for county endpoints?

File: api/can_api_v2_definition.py [hunk 3]
```diff
@@ -132,6 +145,19 @@ class Actuals(base_model.APIBaseModel):
...
+ * capacity - Current staffed ICU bed capacity.
+ * currentUsageTotal - Total number of ICU beds currently in use
+ * currentUsageCovid - Number of ICU beds currently in use by COVID patients.
+""",
+    )
+    hsaIcuBeds: Optional[HospitalResourceUtilization] = pydantic.Field(
+        ...,
+        description="""
+Information about ICU bed utilization details aggregated for the corresponding
+Health Service Area (HSA). For more on HSAs see:
```
Comment (mikelehen) [inline api/can_api_v2_definition.py:158]: Maybe say something about it only being available for county endpoints?

**Review by mikelehen — APPROVED** [review]
Looks great!

File: api/can_api_v2_definition.py (hunk 2 above)
Comment (smcclure17) [reply api/can_api_v2_definition.py]: Good call, added in 49dee09bda7dc6ad9c2f5c9a0c1f2fa37a6dec8c

**Pull request merged**